test/SchemaManager.Core.UnitTests/SchemaManager.Core.UnitTests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
```

Comment:
I am wondering if we should have assemblyName for SchemaManager.Core and SchemaManager.Core.UnitTests to include Microsoft.Health?
<AssemblyName>Microsoft.Health.SchemaManager.Core</AssemblyName>
<AssemblyName>Microsoft.Health.SchemaManager.Core.UnitTests</AssemblyName>

Reply:
It would make sense, I didn't add the prefix since SchemaManager didn't have it either. Should I add it to SchemaManager too? This might be considered a breaking change and require bumping up the major version number on the nuget package.

Reply:
SchemaManager has the assembly name - https://github.com/microsoft/healthcare-shared-components/blob/master/tools/SchemaManager/SchemaManager.csproj#L6